apps/src/blocksCommon.js

```diff
@@ -190,7 +190,7 @@ function installJoinBlock(blockly) {
     },
 
     setInputCount: function (inputCount) {
-      var newInputCount = parseInt(inputCount);
+      var newInputCount = Math.max(parseInt(inputCount), 2);
```

Comment:
it's not clear to me why this is not dangerous; is there no situation in which we want to set inputcount to 1?